drivers/net/ethernet/intel/ice/ice_ethtool_fdir.c

```diff
@@ -1570,7 +1574,7 @@ ice_fdir_write_all_fltr(struct ice_pf *pf, struct ice_fdir_fltr *input,
 		bool is_tun = tun == ICE_FD_HW_SEG_TUN;
 		int err;
 
-		if (is_tun && !ice_get_open_tunnel_port(&pf->hw, &port_num, TNL_ALL))
+		if (is_tun && !ice_get_open_tunnel_port(&pf->hw, TNL_ALL, &port_num))
```

Comment:
Przemek mentioned this on list,. I agree with him that we shouldn't reverse the order of arguments on this function for seemingly no reason. If you just want to align upstream with out-of-tree, we should invert the order out-of-tree.